drivers/lcd/Makefile.include

```diff
@@ -1,4 +1,8 @@
 PSEUDOMODULES += lcd_multi_cntrl
 
+PSEUDOMODULES += lcd_spi
+PSEUDOMODULES += lcd_parallel
```

Comment:
`lcd_parallel_gpio`?
I suppose if we have a native LCD interface we would need a different module name for that.

Reply:
If have already defined a number of further pseudomodules for follow-up PRs that will control which data struct members and what code will be compiled in:

`lcd_parallel` just enables the parallel mode
`lcd_parallel_16bit` will enable 16-bit mode in addition to the 8-bit-parallel mode `lcd_parallel_ll_mcu` will enable the low-level MCU driven interface that has to be implemented by the respective MCU

Reply:
`lcd_parallel` uses the bit-banging implementation if the board doesn't enable the MCU driven implementation by `lcd_parallel_ll_mcu`.

Reply:
To be it would feel cleaner if `lcd_parallel` were just a pseudo-module and each implementation had it's own name. Or is there a reason why you chose to do it that way?

Reply:
`lcd_parallel` is already a pseudomodule which is only intended to enable the parallel interface support, i.e. to declare the required additional members of the device data structure like the GPIOs which are needed independently of the parallel interface implementation and to define functions to use the parallel interface.

Since this PR is only a evolutionary step in a series of follow-on PRs for the final implementation, the GPIO-driven bit-banging implementation is the only implementation for now and is therefore an integral part of the driver. In the final implementation, the GPIO-driven implementation will be moved to a separate file and will be only the default or fallback implementation if no other implementation is enabled.

Instead of trying to explain it in detail, please have a look at the final implementation https://github.com/gschorcht/RIOT-Xtensa-ESP/tree/drivers/lcd_work. Due to its size, I have split it into several steps as separate PRs, so naming the module may not seem reasonable for now.